solver/src/solver/paraswap_solver/api.rs

```diff
@@ -33,7 +36,11 @@ impl ParaswapApi for DefaultParaswapApi {
         let query_str = format!("{:?}", &query);
         let url = query.into_url();
         tracing::debug!("Querying Paraswap API (price) for url {}", url);
-        let response_text = reqwest::get(url)
```

Comment:
nasty leftover spot of not using our shared client (with timeout). Not a big issue since the solver futures have a timeout.